source/clear-linux/concepts/mixer-about.rst

```diff
@@ -3,21 +3,80 @@
 Mixer
 #####
 
-Mixing refers to composing a custom, versioned image of |CL-ATTR| for a
-specific use case. While upstream |CL| provides options to install bundles
-for various capabilities, some developers and OSVs may wish to either
-augment the operating system with functionality from their own packages
-or modify the structure of current bundles. Mixing offers a method to add
-capabilities while retaining the ability to stay up to date with an upstream
-version of |CL|.
-
-Mixing is a multi-step process that starts with installing the mixer bundle
-using swupd. You will also need a working knowledge of
-:abbr:`RPMs (RPM Package Manager files)` and how |CL| bundles work. For a
-detailed guide on how to create a |CL| mix, visit :ref:`mixer`.
+|CL-ATTR| is a powerful, modular, and customizable OS. Upstream |CL| offers
+many images to support different environments and use-cases.  There are
+hundreds of bundles that will meet most, if not all, of your OS and software
+needs.
+
+However, if you need additional customization or content, |CL| provides the
+mixer tool. We provide two different ways to develop and extend the OS, depending on your needs:
```

Comment:
Let's change "However, if you need additional customization or content, |CL| provides the
mixer tool. We provide two different ways to develop and extend the OS, depending on your needs:" to "However, if you need additional customization or content, |CL| provides the
mixer tool. Depending on your needs, the mixer tool allows you to:"  

"develop and extend the OS" seems a little strong for me.

Reply:
OK with me.

Reply:
@bktan8 @mvincerx Updated.